### ISSUE TITLE
Mods affecting rune drop rate?

### ISSUE THREAD

**Issue by jongad** [olegbl/d2rmm.mods#13]
Hi, I've gotten back to the game recently and I'm still using the same mods I used earlier this year. They mostly are cosmetic mods + expanded inventory. I just updated my Mod Manager to from 1.4.3 to 1.5.0.

My High Rune drop rate has been crazy over the past 2 weeks. I found 4 Jah runes and 1 Ber, but I find it really weird since I haven't found anything between Pul and Sur over that time. 

I'm wondering if the mods I have might somehow impact the drop rate.

Here is my mod list:
- Decrease Clutter Drops v1.1
- LightPillar v1.0 (by qhu91it)
- Numbered Rune Names v1.0
- Short Quality Prefixes v1.1
- Custom Item Clors v1.0
- Short Gem Names v1.2
- Short Potion Names 1.1
- Short Scroll Names v1.0
- Skip Intro Videos v1.0
- Expanded Inventory v1.3

Is there a way for me to check if my drop table or probabilities are messed up? I remember when Patch 2.6 arrived got crazy Sunder Charms drops, I had to re-install mods in D2RMM and it fixed the issue.

**Comment by olegbl** [comment]
Rune drop rates are very low. A sample size of 5 means basically nothing.

Assuming you are asking about the Increase Droprates mod, you can check the generated drop table in treasureclassex.txt, as normal.

**Comment by jongad** [comment]
@olegbl Thanks for replying. I understand what you're saying about low probabilities, but I got another Jah rune today. 5th Jah rune in 2 weeks, which is very unusual. The reason I started the issue is I'm wondering if the mods might be affecting drop rates in an unintended way.

I picked those mods explicitly because drop rates are not affected (as I understand it). I checked treasureclassex.txt but apart from swapping some stamina and antidote potions with gold (thanks to Decreased Clutter Drops mod), everything else seems identical to original file. I'm attaching the file for reference.

[treasureclassex.txt](https://github.com/olegbl/d2rmm.mods/files/13768553/treasureclassex.txt)

Is there any other file than this one that can impact drop rates?
